design-documents/message-queue/AWSEventBridge.md

```diff
@@ -0,0 +1,26 @@
+#### 1- AWS EventBridge
```

Comment:
Btw there are already some project, maybe you can reference on it
https://github.com/magento-engcom/amazon-event-bridge